app/frontend/src/features/communities/CommunityPage/CircularIconButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { IconButton, IconButtonProps, makeStyles } from "@material-ui/core";
+import React, { ReactNode } from "react";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    borderRadius: "50%",
+    boxShadow: "0 2px 4px",
+    "&:active": {
+      transform: "translateY(1px)",
+      boxShadow: "0 1px 4px",
+    },
+  },
+}));
+
+interface CircularIconButtonProps extends IconButtonProps {
+  children?: ReactNode;
+  onClick: () => void;
+}
+
+export default function CircularIconButton({
```

Comment:
This component looks a bit out of place. I might be sneaky and change the design later...